### DISCUSSION TITLE
Github support team did not respond my ticket #2070851

### DISCUSSION THREAD

**Discussion by pfuku** [community/community#51226] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

For one week, there is no respond about my ticket. I am waiting a response please!!! My Ticket : #2070851

**Comment by queenofcorgis** [comment, marked as answer]
Hi there - I just checked in on your ticket, and I can confirm it is the right place. Response time is based on the number of tickets ahead of yours and our team's resources. The community is unable to assist or escalate tickets so I'll be locking this post. We appreciate your patience!
